### PR TITLE
Lint code style with Prettier

### PR DESCRIPTION
**Pull request by dmpotter44**
This branch adds Prettier to the lint rules.

### CODE REVIEW

File: src/trialscope.ts [hunk 1]
```diff
@@ -233,7 +250,9 @@ export class TrialScopeQuery {
    * @return {string} the TrialScope GraphQL query
    */
   toQuery(): string {
-    let baseMatches = `conditions:[${Array.from(this.getTrialScopeConditions()).join(', ')}], baseFilters: { zipCode: "${this.zipCode}"`;
+    let baseMatches = `conditions:[${Array.from(this.getTrialScopeConditions()).join(
+      ', '
+    )}], baseFilters: { zipCode: "${this.zipCode}"`;
```
Comment (zlister) [inline src/trialscope.ts:255]: This is a funky one. The line itself is a mess but I'm not sure this makes it any better.

File: src/trialscope.ts [hunk 2]
```diff
@@ -33,24 +35,24 @@ const TRIALSCOPE_PHASES: Record<string, string | null> = {
...
   // Unclear if this is a good mapping
   'in-review': 'NOT_YET_RECRUITING',
   // It's unclear if this mapping is correct
   'temporarily-closed-to-accrual': 'ACTIVE_NOT_RECRUITING',
   'temporarily-closed-to-accrual-and-intervention': 'SUSPENDED',
-  'withdrawn': 'WITHDRAWN',
+  withdrawn: 'WITHDRAWN'
   // NOT MAPPED:
   // ENROLLING_BY_INVITATION
   // TERMINATED
```
Comment (zlister) [inline src/trialscope.ts:58]: I think throwing the prettier-ignore might be worth it here just for consistency.

File: src/bundle.ts [hunk 3]
```diff
@@ -27,19 +27,18 @@ export interface Bundle extends BaseResource {
 }
 
 export function isBundle(o: unknown): o is Bundle {
-  if (typeof o !== 'object')
-    return false;
+  if (typeof o !== 'object') return false;
```
Comment (19ngaddam) [inline src/bundle.ts:30]: i don't like this

Comment (19ngaddam) [reply src/bundle.ts:30]: since there are no braces i guess this makes sense

File: src/trialscope.ts (hunk 1 above)
Comment (dmpotter44) [reply src/trialscope.ts]: I've taken a stab at improving the line to make it more readable.

File: src/trialscope.ts (hunk 2 above)
Comment (dmpotter44) [reply src/trialscope.ts]: I've turned Prettier off for this part.

**Pull request merged**